### PR TITLE
Correct datagrid

### PR DESCRIPTION
**Pull request by fcollonval**
- Use you rather than we
- Use example instead of tutorial
- Update preview
- Rewording

### CODE REVIEW

File: basics/datagrid/README.md [hunk 1]
```diff
@@ -43,11 +49,16 @@ class TutorialView extends StackedPanel {
 }
 ```
 
-That's rather easy. Let's now dive into the `DataModel` class that is taken
-from the official phosphor.js example. The first few lines look like this:
+Your widget is derived from `StackedPanel` to inherit its behavior. Then
+some properties for the panel. Then the `DataGrid` widget and its associated model are created.
+Finally the grid is inserted inside the panel.
+
+The `DataModel` class is not used directly as its an abstract class.
```
Comment (jtpio) [inline basics/datagrid/README.md]:
```suggestion
The `DataModel` class is not used directly as it's an abstract class.
```

File: basics/datagrid/README.md [hunk 2]
```diff
@@ -79,19 +92,17 @@ type ColumnRegion = 'body' | 'row-header';
...
 
-The meaning of these lines might be obvious for experienced users of typescript
-or Haskell. The `|` can be read as or. This means that the `RowRegion` type is
-either `body` or `column-header`, explaining what the `rowCount` and
-`columnCount` functions do: They define a table with `2` header rows, with 3
-index columns, with `1000000000000` rows and `1000000000000` columns.
+The `|` can be read as or. This means that the `RowRegion` type is
+either `body` or `column-header`.
+
+So the `rowCount` and `columnCount` functions define a table with `2` header rows, with 3 index columns, with `1000000000000` rows and `1000000000000` columns.
```
Comment (jtpio) [inline basics/datagrid/README.md]: "3" could be quoted to keep the same style as for the other numbers:

```suggestion
So the `rowCount` and `columnCount` functions define a table with `2` header rows, with `3` index columns, with `1000000000000` rows and `1000000000000` columns.
```

File: basics/datagrid/README.md [hunk 3]
```diff
@@ -79,19 +92,17 @@ type ColumnRegion = 'body' | 'row-header';
...
 
-The remaining part of the LargeDataModel class defines the data values of the
-datagrid. In this case it simply displays the row and column index in each
-cell, and adds a letter prefix in case that we are in any of the header
-regions:
+Finally the `data` method of `LargeDataModel` class defines the data
```
Comment (jtpio) [inline basics/datagrid/README.md]:
```suggestion
Finally the `data` method of the `LargeDataModel` class defines the data
```

File: basics/datagrid/src/index.ts [hunk 4]
```diff
@@ -2,17 +2,13 @@ import {
...
 } from '@jupyterlab/application';
-
-import { IMainMenu } from '@jupyterlab/mainmenu';
-
-import { Menu } from '@phosphor/widgets';
-
 import { ICommandPalette } from '@jupyterlab/apputils';
-
-import { StackedPanel } from '@phosphor/widgets';
+import { IMainMenu } from '@jupyterlab/mainmenu';
```
Comment (jtpio) [inline basics/datagrid/src/index.ts]: Should we keep an empty line between imports?

File: basics/datagrid/src/index.ts [hunk 5]
```diff
@@ -26,34 +22,34 @@ const extension: JupyterFrontEndPlugin<void> = {
...
-        const widget = new TutorialView();
+      execute: () => {
+        const widget = new ExampleView();
         shell.add(widget, 'main');
       }
     });
     palette.addItem({ command, category });
 
-    let tutorialMenu: Menu = new Menu({ commands });
+    let exampleMenu: Menu = new Menu({ commands });
```
Comment (jtpio) [inline basics/datagrid/src/index.ts]: Maybe we can make this variable `const`?

File: basics/datagrid/src/index.ts (hunk 4 above)
Comment (fcollonval) [reply basics/datagrid/src/index.ts]: You are right, it helps readability (side effect of the automatic sort import algorithm).

File: basics/datagrid/src/index.ts (hunk 5 above)
Comment (fcollonval) [reply basics/datagrid/src/index.ts]: Done, I also remove the typing as Typescript will solve it easily.

**Comment by fcollonval** [comment]
Thanks @jtpio for the review

**Pull request merged**